client/js/lib/core/loadLibs.js

```diff
@@ -10,30 +10,35 @@
 // Dp libs -> initialize on mount of vue instance
 import {
   ActionMenu,
+  Confirm,
+  Sticky
+} from '@demos-europe/demosplan-utils'
+
+import {
+  addFormHiddenField,
```

Comment:
Both `addFormHiddenField` and `removeFormHiddenField` do not have to be imported or invoked here. They are only called in specific places.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/666/commits/61c2dcceef5ba7f21b610171cc73eb5779b3b953